py/client/pydeephaven/table.py

```diff
@@ -804,3 +804,79 @@ def delete(self, table: Table) -> None:
             self.session.input_table_service.delete(self, table)
         except Exception as e:
             raise DHError("delete data in the InputTable failed.") from e
+
+
+class MultiJoinTable:
+    """A MultiJoinTable is an object that contains the result of a multi-table natural join. To retrieve the underlying
+    result Table, use the table() method. """
+
+    def __init__(self, table: Table):
+        self._table = table
+
+    def table(self) -> Table:
```

Comment:
This is consistent between the client and server.  I want to point out that we do have some API stuff that feels a little inconsistent.  For example, here `.table()`  must be called.  For `PartitionedTableProxy`, `.target` is called, without the `()`.  We need some religion about when we do or don't use `()` in the API.

Reply:
It looks like this went into v0.36.1 via #4356.  It has not been in long, so we would break far less people if we change now vs later... but I hate breaking people, and I hate inconsistent APIs.